qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -254,6 +255,72 @@ def defaults(self, refresh: bool = False) -> Optional[PulseDefaults]:
 
         return self._defaults
 
+    def job_limit(self) -> BackendJobLimit:
+        """Return job limit for the backend.
+
+        The job limit information may include the current number of
+        jobs running on the backend at this time and the maximum number
+        of concurrent jobs a user could submit to the backend at a time.
+
+        Note:
+            The job limit information for the backend is provider specific.
+            For example, if you have access to the same backend via
+            different providers, the job limit information might be
+            different for each provider.
+
+            If the method call was successful, you can inspect the job
+            limit for the backend by accessing the ``maximum_jobs``
+            and ``running_jobs`` attributes of the ``BackendJobLimit``
+            instance returned.
+
+            For example:
+                backend_job_limit = backend.job_limit()
+                maximum_jobs = backend_job_limit.maximum_jobs
+                running_jobs = backend_job_limit.running_jobs
+
+            * If ``maximum_jobs`` or ``running_jobs`` are ``None``, the
+                job limit information is currently not available.
+            * If ``maximum_jobs`` is equal to ``-1``, then there are
+                no limits to the maximum number of concurrent jobs a user
+                could submit to the backend at a time.
+
+        Returns:
+            the job limit for the backend with this provider.
+
+        Raises:
+            LookupError: If jobs limit for the backend can't be found.
+        """
+        api_job_limit = self._api.backend_job_limit(self.name())
+
+        try:
+            return BackendJobLimit.from_dict(api_job_limit)
+        except ValidationError as ex:
+            raise LookupError(
+                "Couldn't get backend jobs limit: {0}".format(ex))
+
+    def remaining_jobs_count(self) -> int:
+        """Return remaining jobs that could be submitted to the backend.
```

Comment:
nit: can you update the first sentence to include `the number`? since only the first line in the docstring is displayed in the summary table.